components/library/qt5/qt5.p5m

```diff
@@ -0,0 +1,10273 @@
+file path=usr/lib/qt/5.8/doc/global/template/images/arrow_down.png
+file path=usr/lib/qt/5.8/doc/global/template/images/bg_l.png
+file path=usr/lib/qt/5.8/doc/global/template/images/bg_l_blank.png
+file path=usr/lib/qt/5.8/doc/global/template/images/bg_ll_blank.png
+file path=usr/lib/qt/5.8/doc/global/template/images/bg_r.png
+file path=usr/lib/qt/5.8/doc/global/template/images/bg_ul_blank.png
+file path=usr/lib/qt/5.8/doc/global/template/images/bgrContent.png
+file path=usr/lib/qt/5.8/doc/global/template/images/blu_dot.png
+file path=usr/lib/qt/5.8/doc/global/template/images/box_bg.png
+file path=usr/lib/qt/5.8/doc/global/template/images/breadcrumb.png
+file path=usr/lib/qt/5.8/doc/global/template/images/btn_next.png
+file path=usr/lib/qt/5.8/doc/global/template/images/btn_prev.png
+file path=usr/lib/qt/5.8/doc/global/template/images/bullet_dn.png
+file path=usr/lib/qt/5.8/doc/global/template/images/bullet_gt.png
+file path=usr/lib/qt/5.8/doc/global/template/images/bullet_sq.png
+file path=usr/lib/qt/5.8/doc/global/template/images/bullet_up.png
+file path=usr/lib/qt/5.8/doc/global/template/images/feedbackground.png
+file path=usr/lib/qt/5.8/doc/global/template/images/header_bg.png
+file path=usr/lib/qt/5.8/doc/global/template/images/home.png
+file path=usr/lib/qt/5.8/doc/global/template/images/horBar.png
+file path=usr/lib/qt/5.8/doc/global/template/images/ico_note.png
+file path=usr/lib/qt/5.8/doc/global/template/images/ico_note_attention.png
+file path=usr/lib/qt/5.8/doc/global/template/images/ico_out.png
+file path=usr/lib/qt/5.8/doc/global/template/images/logo.png
+file path=usr/lib/qt/5.8/doc/global/template/images/page.png
+file path=usr/lib/qt/5.8/doc/global/template/images/page_bg.png
+file path=usr/lib/qt/5.8/doc/global/template/images/spinner.gif
+file path=usr/lib/qt/5.8/doc/global/template/images/sprites-combined.png
+file path=usr/lib/qt/5.8/doc/global/template/scripts/extras.js
+file path=usr/lib/qt/5.8/doc/global/template/scripts/main.js
+file path=usr/lib/qt/5.8/doc/global/template/style/cookiebar-x.png
+file path=usr/lib/qt/5.8/doc/global/template/style/doc_search.png
+file path=usr/lib/qt/5.8/doc/global/template/style/gsc.css
+file path=usr/lib/qt/5.8/doc/global/template/style/icomoon.eot
+file path=usr/lib/qt/5.8/doc/global/template/style/icomoon.svg
+file path=usr/lib/qt/5.8/doc/global/template/style/icomoon.ttf
+file path=usr/lib/qt/5.8/doc/global/template/style/icomoon.woff
+file path=usr/lib/qt/5.8/doc/global/template/style/list_arrow.png
+file path=usr/lib/qt/5.8/doc/global/template/style/list_expand.png
+file path=usr/lib/qt/5.8/doc/global/template/style/offline-simple.css
+file path=usr/lib/qt/5.8/doc/global/template/style/offline.css
+file path=usr/lib/qt/5.8/doc/global/template/style/online.css
+file path=usr/lib/qt/5.8/doc/global/template/style/theqtcompany.png
+file path=usr/lib/qt/5.8/include/Qt3DCore/$(COMPONENT_VERSION)/Qt3DCore/private/aspectcommanddebugger_p.h
```

Comment:
I have two builds with 32 and 64 bits on alarcher's computer that are slightly different. I need to double check that. It's about those config files. I'll come back as soon as I'll have checked.